plugins/services/src/js/pages/task-details/__tests__/TaskDetail-test.js

```diff
@@ -19,13 +19,18 @@ const TestUtils = require('react-addons-test-utils');
 /* eslint-disable no-unused-vars */
 const MesosSummaryStore = require('../../../../../../../src/js/stores/MesosSummaryStore');
 /* eslint-enable no-unused-vars */
+const DCOSStore = require('foundation-ui').DCOSStore;
 const MesosStateStore = require('../../../../../../../src/js/stores/MesosStateStore');
 const Task = require('../../../structs/Task');
 const TaskDirectory = require('../../../structs/TaskDirectory');
 const TaskDirectoryStore = require('../../../stores/TaskDirectoryStore');
 const TaskDetail = require('../TaskDetail');
 
 describe('TaskDetail', function () {
+  DCOSStore.serviceTree = {
+    findItemById: jest.fn()
```

Comment:
Since you are mocking this function, would it make sense checking if it's called?